app/models/customer.py

```diff
@@ -0,0 +1,27 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+from sqlalchemy.orm import backref, relationship
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable = True)
+    videos_checked_out_count = db.Column(db.Integer, default = 0) 
```

Comment:
Instead of `videos_checked_out_count` being a column we can use the rentals to count that.  

To do that it's helpful to add a relationship:
```rentals = db.relationship('Rental', back_populates='customer', lazy=True)```

For `to_dict`, once you have the relationship the `videos_checked_out_count` is the length of the `rentals`:
```videos_checked_out_count": len(self.rentals)```

This method of dynamically computing `videos_checkout_out_count` rather than storing it in an instance variable makes it so there isn’t information stored in multiple places that could potentially conflict.